sesame/src/main/scala/org/w3/banana/sesame/SesameHttpSparqlEngine.scala

```diff
@@ -0,0 +1,57 @@
+package org.w3.banana.sesame
```

Comment:
General comments on this file, basically try and emulate the [jena version](https://github.com/banana-rdf/banana-rdf/blob/series/0.8.x/jena/src/main/scala/org/w3/banana/jena/JenaSparqlHttpEngine.scala) :

* Don't use the sesame types directly, eg rather `Sesame#Node` than `Value`
* Jena uses `QueryExecutionFactory.sparqlService` so you need to find the sesame equivalent to make the rest of the code like the jena one